examples/40_advanced/README.txt

```diff
@@ -0,0 +1,10 @@
+.. _advanced_examples:
+
+=================
+Advanced Examples
+=================
+
+Examples on customizing Auto-sklearn to other ones use case by changing the
```

Comment:
Hi Matthias, I don't understand the 'to other ones use case'. Do you think it can be re-phrased?

Reply:
That was an oversight, should have been 'to ones use case'